wiki/Active-Effect-Guide.md

```diff
@@ -0,0 +1,440 @@
+# FoundryVTT dnd5e Active Effects Examples
+
+![Up to date as of 2.1.x](https://img.shields.io/badge/dnd5e-v2.1.x-informational)
```

Comment:
This seems old

Reply:
Yea, this just came from Calego's original document, but 2.1 was when the access to the different parts of a Dice Scale Value was added, so I think that 2.1 represents the last time a change was made to the relevant data here, but I can bump it to 2.3 or 2.4 to represent the release of when the document was updated

Reply:
Yeah I think since this information is up-to-date as of 2.3 it's fine to bump this up.